tests/context.py

```diff
@@ -0,0 +1,50 @@
+from tests.java_definitions import *
```

Comment:
I'm not a big fan of the * import.  How about:

```
import tests.java_definitions as java
...
sql_to_substrait = java.SqlToSubstraitClass()
```

Or you could do `isthmus` instead of `java`:

```
import tests.java_definitions as isthmus
...
sql_to_substrait = isthmus.SqlToSubstraitClass()
```